doc/tutorial/Tutorial.tex

```diff
@@ -410,18 +405,8 @@ \section{Introduction to the proof engine: Minimal Logic}
 and go back to Coq's main loop. We may also use \verb:Restart: to restart
```

Comment:
I'm discovering this tutorial today. I think this entire part should be dropped.  Today to restart a proof you go back to be beginning of it, you don't Restart/Reset.  You don't write Undo, you do undo.